x11-misc/gpaste/gpaste-3.36.3.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION="0.30"
+VALA_USE_DEPEND="vapigen"
+
+inherit meson vala
+
+DESCRIPTION="Clipboard management system"
+HOMEPAGE="https://github.com/Keruspe/GPaste"
+SRC_URI="https://github.com/Keruspe/GPaste/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="introspection gnome vala"
+REQUIRED_USE="vala? ( introspection )"
+
+DEPEND="
+	vala? ( $(vala_depend) )
+	virtual/pkgconfig
+"
+RDEPEND="${DEPEND}
+	gnome? (
+		gnome-base/gnome-shell
+	)
+"
+BDEPEND="
+	dev-libs/appstream-glib
+	>=dev-libs/glib-2.48:2
+	>=dev-libs/gobject-introspection-1.48.0
+	sys-apps/dbus
+	>=x11-libs/gdk-pixbuf-2.34:2
+	>=x11-libs/gtk+-3.20:3
+	x11-libs/libX11
+	x11-libs/libXi
+	gnome? (
+		gnome-base/gnome-control-center:2
+		media-libs/clutter
+		x11-libs/pango
+	)
+"
+S=${WORKDIR}/GPaste-${PV}
+
+src_prepare() {
+	use vala && vala_src_prepare
+	default
+}
+
+src_configure() {
+	local emesonargs=(
+		-Dsystemd=true
+		-Dbash-completion=true
+		-Dzsh-completion=true
+		-Dx-keybinder=true
```

Comment:
I believe this flag will make it depend on *gnome-keybindings* whatever package provides that.

```
C linker for the host machine: x86_64-pc-linux-gnu-gcc ld.bfd 2.34
Host machine cpu family: x86_64
Host machine cpu: x86_64
Found pkg-config: /usr/bin/x86_64-pc-linux-gnu-pkg-config (1.7.3)
Run-time dependency appstream-glib found: YES 0.7.17
Run-time dependency gdk-3.0 found: YES 3.24.22
Run-time dependency gdk-pixbuf-2.0 found: YES 2.40.0
Run-time dependency gio-2.0 found: YES 2.64.5
Run-time dependency glib-2.0 found: YES 2.64.5
Run-time dependency gobject-2.0 found: YES 2.64.5
Run-time dependency gtk+-3.0 found: YES 3.24.22
Run-time dependency pango found: YES 1.42.4
Run-time dependency dbus-1 found: YES 1.12.20
Found CMake: /usr/bin/cmake (3.18.2)
Run-time dependency gnome-keybindings found: NO (tried pkgconfig and cmake)

meson.build:34:2: ERROR: Dependency "gnome-keybindings" not found, tried pkgconfig and cmake

A full log can be found at /var/tmp/portage/x11-misc/gpaste-3.36.3/work/gpaste-3.36.3-build/meson-logs/meson-log.txt
 * ERROR: x11-misc/gpaste-3.36.3::testworld failed (configure phase):
```
You can straight DEPEND on it, or make it some sort of *keybinds* USE flag maybe.

Reply:
Resolved by including x-keybinder only when gnome use flag is is activated.